tests/rundoctest.py

```diff
@@ -8,12 +8,18 @@
 path = os.path.dirname(__file__)
 if path:
     os.chdir(path)
+os.chdir('..')
```

Comment:
This line is dangerous! It will change the behavior of `import cypari2`... you would better first test cypari2 as it used to be and in a second time, go to the appropriate directory and doctest autogen.